### PR TITLE
OSSMDOC-644: Update service mesh rel notes for Kiali version

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.6-4.11

Issue: https://issues.redhat.com/browse/OSSMDOC-644

Link to docs preview: http://file.bos.redhat.com/tokeefe/OSSMDOC-644/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-version-2-2-1

Additional information:

Dev SME Jay Shaughnessy 
QE Filip Brychta.
Peer Review: jstickler

### CODE REVIEW

File: modules/ossm-rn-new-features.adoc [hunk 1]
```diff
@@ -34,7 +34,7 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 |1.34.1
 
 |Kiali
-|1.48.1
+|1.48.2.1
```
Comment (jshaughn) [inline modules/ossm-rn-new-features.adoc:37]:
```suggestion
|1.48.2-1
```

File: modules/ossm-rn-new-features.adoc [hunk 2]
```diff
@@ -124,7 +124,7 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 |1.29.1
 
 |Kiali
-|1.36.11
+|1.36.12.1
```
Comment (jshaughn) [inline modules/ossm-rn-new-features.adoc:127]:
```suggestion
|1.36.12-1
```

File: modules/ossm-rn-new-features.adoc [hunk 3]
```diff
@@ -352,7 +352,7 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 |1.28.0
 
 |Kiali
-|1.24.15-1
+|1.24.16.1
```
Comment (jshaughn) [inline modules/ossm-rn-new-features.adoc:355]:
```suggestion
|1.24.16-1
```

**Review by jshaughn — CHANGES_REQUESTED** [review]
I believe the container versions are dash1 as opposed to dot1.

**Comment by FilipB** [comment]
I'm wondering if it's necessary to include the last part. Is it accessible to users anywhere? Kiali operator is shown as 1.48.2 in OCP console and Kiali about dialog also shows 1.48.2.

**Comment by rh-tokeefe** [comment]
@FilipB that's a good question. On prior releases, we tended to list the fourth value for additional clarity, even though the console does not show it.

**Review by jshaughn — APPROVED** [review]

File: modules/ossm-rn-new-features.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: Done

File: modules/ossm-rn-new-features.adoc (hunk 2 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: Done

File: modules/ossm-rn-new-features.adoc (hunk 3 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc]: Done

**Comment by FilipB** [comment]
LGTM

**Review by JStickler — APPROVED** [review]
LGTM

**Pull request merged**